lib/pierre.py

```diff
@@ -157,9 +160,25 @@ def get_bodies(event_object):
     return bodies
 
 
+def has_pull_request(event):
+    if "pull_request" in event:
+        return True
+    elif "issue" in event:
+        return "pull_request" in event.get("issue")
+    else:
+        return False
+
+
 def get_pull_request_id(event):
-    logger.info(f'Pull Request ID: {event.get("issue", {}).get("number", "")}')
-    return event.get("issue", {}).get("number", "")
+    if "pull_request" in event:
+        pr_id = event.get("pull_request").get("number")
```

Comment:
PR `number` for opened PR events resides under `pull_request`, not under `issue`.
https://developer.github.com/webhooks/event-payloads/#pull_request